Documentation/Settings.md

```diff
@@ -297,22 +291,19 @@ Display brightness. Higher values age the OLED faster due to burn-in. (However,
 
 On device help text:
 
-Adjust OLED screen brightness
+Adjust the OLED screen brightness
 
 ### Setting: Invert screen
 
 Inverts the entire OLED.
 
 On device help text:
 
-Invert OLED screen colours
+Invert the OLED screen colors
 
 ### Setting: Boot logo duration
 
-Sets the duration of the boot logo from 1 - 4 seconds.
-- For static images this sets the time the logo is displayed for.
-- For animations this sets the time the last frame is displayed for after the animation has been played.
-- The infinity $∞$ icon sets a logo or the last frame of an animation to be displayed until a button is pressed.
+Sets the duration for the boot logo (S=seconds).
```

Comment:
S should be lowercase

Reply:
Fixed in [307b6c7](https://github.com/Ralim/IronOS/commit/307b6c76f8d6cb42ad59d5e91729fdce0caee773)